CMakeLists.txt

```diff
@@ -7,13 +7,19 @@ project(VulkanBootstrap
     DESCRIPTION "A Vulkan utility library to ease the initialization steps in Vulkan"
     VERSION ${VK_BOOTSTRAP_SOURCE_HEADER_VERSION})
 
+# Occasionally Options DO NOT WORK since they don't get updated in the cached cmake files
```

Comment:
Is this comment needed now that the option to build as a shared library is using the BUILD_SHARED_LIBS option?